kyo-prelude/shared/src/main/scala/kyo2/Choice.scala

```diff
@@ -0,0 +1,32 @@
+package kyo2
+
+import kyo.Tag
+import kyo2.kernel.*
+
+sealed trait Choice extends Effect[Seq, Id]
+
+object Choice:
+
+    inline def get[T](seq: Seq[T]): T < Choice =
+        Effect.suspend[T](Tag[Choice], seq)
+
+    inline def eval[T, U, S](seq: Seq[T])(inline f: T => U < S): U < (Choice & S) =
+        seq match
+            case Seq(head) => f(head)
+            case seq       => Effect.suspend[T](Tag[Choice], seq, f)
+
+    inline def dropIf[S](condition: Boolean): Unit < (Choice & S) =
```

Comment:
A minor API change renaming this method from `filter` to `dropIf`. It seems more intuitive, `filter` made more sense when the effect was still named `Seqs`.

Reply:
No, it's probably a leftover. Good catch!